OpenSim/Common/APDMDataReader.cpp

```diff
@@ -119,8 +123,8 @@ APDMDataReader::extendRead(const std::string& fileName) const {
     bool foundMagneticHeadingData = magIndex.size()>0;
     bool foundAngularVelocityData = gyroIndex.size()>0;
 
-    // If no Orientation data is available or dataRate can't be deduced we'll abort completely
-    OPENSIM_THROW_IF((orientationsIndex.size() == 0 || SimTK::isNaN(dataRate)),
+    // If no Orientation data is available we'll abort
```

Comment:
It should be legitimate IMU file that does not have orientation data but has linear acceleration, angular velocity and/or magnetic heading

Reply:
@aseth1 In the two examples format we have we always had orientations while other data was missing, in fact orientation is needed for downstream usage AFAIK. If you prefer we don't abort here and abort downstream on demand let me know.